lib/controller.js

```diff
@@ -113,7 +114,8 @@ Tessel.get = function(opts) {
     var seekerOpts = {
       timeout: (opts.timeout || 2) * 1000,
       usb: opts.usb,
-      lan: opts.lan
+      lan: opts.lan,
+      authorized: opts.authorized || true
```

Comment:
I recommend passing through the option as is. Then, inside the [seeker.start](https://github.com/tessel/t2-cli/blob/master/lib/discover.js#L26-L47) function, you can sanitize the option. I don't think you need should set it to true by default.